strawberryfields/logger.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains functions for creating a logger that can be used in Strawberry
+Fields.
+
+The implementation in this module is based on the solution for logging used in
+the Flask web application framework:
+https://github.com/pallets/flask/blob/master/src/flask/logging.py
```

Comment:
Just to clarify, no portions of the code from the flask module above were used directly, were they? If they were, we will need to provide proper attribution

Reply:
Thanks for this! I'll update it such that we provide proper attribution here as we discussed